src/components/ExpansionPanel/Body/index.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Body = props => <div {...props} />;
+
+Body.propTypes = {
+  /**
+   * The content to render.
+   */
+  children: PropTypes.node,
```

Comment:
Since you're just passing the props, you don't need to add the propTypes anymore.